src/moscot/backends/ott/_jax_data.py

```diff
@@ -0,0 +1,137 @@
+from functools import partial
+from types import MappingProxyType
+from typing import Any, Dict, List, Literal, Tuple, Union
+
+import jax
+import jax.numpy as jnp
+from ott.geometry.pointcloud import PointCloud
+from ott.problems.linear import linear_problem
+from ott.solvers.linear import sinkhorn
+
+
+class JaxSampler:
+    """Data sampler for Jax."""
+
+    def __init__(
+        self,
+        distributions: List[jnp.ndarray],  # type: ignore[name-defined]
```

Comment:
I'd just use a list of `linear_problem.LinearProblem()`, this way, you can absorb the following arguments:
- distributions (contained in the `.geom` as a point cloud)
- a
- b
- tau_a
- tau_b
- epsilon (contained in the `.geom`)
- policy_pairs - can be constructed outside (or, e.g., have a `from_policy` class method).